src/box/txn_limbo.c

```diff
@@ -176,16 +176,22 @@ txn_limbo_append(struct txn_limbo *limbo, uint32_t id, struct txn *txn)
 }
 
 static inline void
-txn_limbo_remove(struct txn_limbo *limbo, struct txn_limbo_entry *entry)
+txn_limbo_pop_confirmed_entry(struct txn_limbo *limbo,
+			      struct txn_limbo_entry *entry, bool is_async)
 {
 	assert(!rlist_empty(&entry->in_queue));
 	assert(txn_limbo_first_entry(limbo) == entry);
+	(void)is_async;
```

Comment:
Do we really wanna add a function argument just to pass it to the assertion? I don't like that a lot